docs/source/conf.py

```diff
@@ -321,6 +321,6 @@
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
     'python': ('https://docs.python.org/3.6', None),
-    'jinja': ('http://jinja.pocoo.org/docs/dev', None),
+    'jinja': ('https://jinja.palletsprojects.com/en/latest/', None),
```

Comment:
I suspect this was failing because the URL was a redirect and builds don't like redirects. maybe it was just a front end redirect which can cause problems. the url above is correct.